activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -916,6 +920,13 @@ def set_value(name, value) # :nodoc:
 
     private
 
+      # Extract column names from arguments passed to #order or #reorder.
+      def column_names_from_order_arguments(args)
+        args = args.dup
+        orderings = args.extract_options!
```

Comment:
passing `*args` (`column_names_from_order_arguments(*args)`) and defining `def column_names_from_order_arguments(*args, **orderings)`.